src/LiveComponent/src/Hydration/AbstractDoctrineHydrationExtension.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\LiveComponent\Hydration;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\Persistence\ManagerRegistry;
+use Doctrine\Persistence\ObjectManager;
+
+/**
+ * @author Jean-Paul van der Wegen <info@jpvdw.nl>
```

Comment:
Maybe the original author of the methods ?

Reply:
There was no original author. i remove this comment.

Reply:
Oh sorry it's because you extract a bit of code from the DoctrineHydrationExtension and the way github presented me induced me in error